### ISSUE TITLE
dlsym issue when compiling

### ISSUE THREAD

**Issue by adriencarbonaro** [powerman/wcwidth-icons#4]
This is my output for `make` and `sudo make install`

```
cc -ldl -Wl,-z,defs -Wl,--as-needed -Wl,-soname,libwcwidth-icons.so -shared -fPIC -o libwcwidth-icons.so wcwidth-icons.c
/tmp/cckz5NOG.o : In function « wcwidth » :
wcwidth-icons.c:(.text+0x26) : undefined reference to « dlsym »
collect2: error: ld returned 1 exit status
Makefile:6: recipe for target 'libwcwidth-icons.so' failed
make: *** [libwcwidth-icons.so] Error 1
```

Do you know why ?

**Comment by powerman** [comment]
Not sure, I suppose this may be related to compiler version. I use gcc-7.3.0 and don't have this error. You can check this answer https://stackoverflow.com/a/20419191/113120 and see is it helps.

**Comment by mpfaff** [comment]
Hi. I've tried the solution in the link provided but I am still facing the issue.

**Comment by powerman** [comment]
@adriencarbonaro Did you've solved the issue?

**Comment by kronolynx** [comment]
I had the same problem with gcc (Ubuntu 8.3.0-6ubuntu1) 8.3.0

```
cc -ldl -Wl,-z,defs -Wl,--as-needed -Wl,-soname,libwcwidth-icons.so -shared -fPIC -o libwcwidth-icons.so wcwidth-icons.c
/usr/bin/ld: /tmp/ccYxkP5j.o: in function `wcwidth':
wcwidth-icons.c:(.text+0x26): undefined reference to `dlsym'
collect2: error: ld returned 1 exit status
make: *** [Makefile:6: libwcwidth-icons.so] Error 1
```

I tried as suggested in the link and it compiles without problems
```bash
LDFLAGS += -Wl,-z,defs -Wl,--no-as-needed

libwcwidth-icons.so: wcwidth-icons.c
	$(CC) $(LDFLAGS) $(CFLAGS) -Wl,-soname,libwcwidth-icons.so -shared -fPIC -o $@ $^
```

**Comment by adriencarbonaro** [comment]
I'll try this when I get a chance

**Comment by kronolynx** [comment]
I'm probably doing something wrong

A regular nerd font doesn't display properly
`URxvt.font: xft:FantasqueSansMono Nerd Font:size=12`
![double](https://i.imgur.com/SPaiYxK.png)

with a mono font: 
`URxvt.font: xft:FantasqueSansMono Nerd Font Mono:size=12`
![mono](https://i.imgur.com/bgn0baS.png)

**Comment by powerman** [comment]
Fixed in v0.2.2.
